### ISSUE TITLE
XEPs support page with version

### ISSUE THREAD

**Issue by Neustradamus** [dino/dino#308]
On the XEPs support page, it is possible to add version of XEPs?
https://github.com/dino/dino/wiki/Supported-XEPs

Examples:
- https://metronome.im/supported-xeps-list
- https://trac.gajim.org/wiki/GajimXEPSupport
- https://github.com/jappix/jappix/blob/master/PROTOCOL.md

**Comment by mightyBroccoli** [comment]
https://dev.gajim.org/gajim/gajim/wikis/help/GajimXEPSupport

This is the correct url for the Gajim XEP List btw.

**Comment by Neustradamus** [comment]
Good point @mightyBroccoli, there was a change on the new wiki, thanks for the information :)
